samples/angular9-sample-app/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { TestBed, async } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatButtonModule } from '@angular/material/button';
+import { MatListModule } from '@angular/material/list';
+import { MsalService, MSAL_CONFIG, MSAL_CONFIG_ANGULAR, MsalAngularConfiguration, BroadcastService } from '@azure/msal-angular';
+import { Configuration } from 'msal';
+
+describe('AppComponent', () => {
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule,
+        MatToolbarModule,
+        MatButtonModule,
+        MatListModule,
+      ],
+      declarations: [
+        AppComponent
+      ],
+      providers: [
+        MsalService,
+        {
+          provide: MSAL_CONFIG,
+          useValue: {
+            auth: {
+              clientId: 'Enter_the_Application_Id_here', // This is your client ID
+              authority: 'https://login.microsoftonline.com/Enter_the_Tenant_Info_Here', // This is your tenant info
+              redirectUri: 'Enter_the_Redirect_Uri_Here' // This is your redirect URI
```

Comment:
if the intent is to have the sample working out of the box, do we also want these prefilled with the provided sample info? or should both be left blank? It feels odd that we're instructing to put this information here but not in the actual config in `app.module.ts`, for example

Reply:
Yeah, the samples in the repo should have the clientId filled in. Not a big deal in this specific file, since its a test and doesn't actually need valid info, though.

Reply:
okay, cool--I was mostly confused when I was reading this test because the information made it look like it would need to be filled out in order for it to work.